### DISCUSSION TITLE
Python2 version

### DISCUSSION THREAD

**Discussion by fatshotty** [miguelgrinberg/Flask-SocketIO#1636] [General]
Hello
I don't know if this is the right place...
I need to use the last version of socket-io[client] (engineio etc..) compatible with python2. My socket-io-server is 4.1.2; socket-io python3 client library is working very well, but I need to keep compatibility with python2

Thanks in advance

**Comment by miguelgrinberg** [comment]
Use the same versions of python-socketio and python-engineio that you have on the server side.
